spec/unit/sneakers_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+require 'sneakers'
+require 'airbrake/sneakers'
+
+RSpec.describe Airbrake::Sneakers::ErrorReporter do
+  let(:worker) do
+    queue = instance_double('queue')
+    allow(queue).to receive_messages(
+      name: 'test-queue',
+      opts: {},
+      exchange: instance_double('exchange')
+    )
+
+    Sneakers.clear!
```

Comment:
We should probably call this in an `after` block. Last executed test will leave Sneakers "dirty".

Reply:
true